clang/include/clang/CIR/Interfaces/ASTAttrInterfaces.td

```diff
@@ -0,0 +1,213 @@
+//===- ASTAttrInterfaces.td - CIR AST Interface Definitions -----*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef MLIR_CIR_INTERFACES_AST_ATTR_INTERFACES
+#define MLIR_CIR_INTERFACES_AST_ATTR_INTERFACES
+
+include "mlir/IR/OpBase.td"
+
+let cppNamespace = "::mlir::cir" in {
+
+    def ASTDeclInterface : AttrInterface< "ASTDeclInterface" > {
```

Comment:
Since clang-format won't touch these files, please remove the spaces around `<`, `>`, so `AttrInterface< "ASTDeclInterface" >` will become `AttrInterface<"ASTDeclInterface">`, etc.